docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+<!--
+---
+linkTitle: "Chains:v2"
+weight: 10
+---
+-->
+
+# Build Type
+
+This buildType is maintained by [TektonCD/Chains](https://github.com/tektoncd/chains) and describes a build executed by [TektonCD/Pipelines](https://github.com/tektoncd/pipeline).
+
+
+# Description
+
+```
+"buildType": "https://tekton.dev/predicate/chains/slsa/v2"
+```
+
+This buildType describes the execution of a build via a `PipelineRun` or a `TaskRun`.
```

Comment:
Could we elaborate more on the definition of the build type? or pointing to slsa definition of the build type https://slsa.dev/spec/v1.0/provenance#buildType?

Reply:
added a link.